### ISSUE TITLE
Internationalization with Weblate

### ISSUE THREAD

**Issue by FlorentRevest** [AsteroidOS/WinSteroid#1]
Since WinSteroid is an upstream project, it should be available to translation on Weblate: https://hosted.weblate.org/projects/asteroidos/
I am not familiar with internationalization of Windows apps so I'm afraid I won't be very helpful here. Maybe @TheLastProject would have some input on this?

**Comment by TheLastProject** [comment]
With a quick look through the source code, I noticed some .NET things, so my best guess would be "Resource files" (https://docs.weblate.org/en/latest/formats.html#net-resource-files). I'm not a Windows developer though, but it seems most hopeful :)

**Comment by thunderluca** [comment]
I confirm the resource files, but with a twist: Universal Windows Platform doesn't use classic .NET .resx files, but .resw, that are simpler than .resx files (because .resw files are rebranded xml only files, .resx instead generates a .cs file), but I don't see in Weblate any reference to .resw files. I'll try to use both file formats, in case .resw files are not supported.

But I hoped to extract a resw template, because in UWP you "only" need to put a Resources.resw file in a folder under Strings directory with 2-letters ISO language as parent folder name, and Windows 10 will automatically choose the Resource file to use by current UI thread user language (and in case it doesn't found a matching resource key for that language, it automatically use en-us language as fallback (specified in package manifest default language).

P.S.: It's better if someone can also review the current en-US resource file I have created, because of my bad English level :-)

**Comment by FlorentRevest** [comment]
Thanks for the investigation @TheLastProject and @thunderluca !
I reviewed your en-US translation and pushed a commit correcting various strings I was pretty confident needed changes https://github.com/AsteroidOS/WinSteroid/commit/486ba130ccd57b02fd2fb0dd14ee18df43345cf7

**Comment by thunderluca** [comment]
Hi guys,

I discovered that UWP applications don't like old .resx files, especially for new UI references by x:Uid that were not supported but only by .resw files. 

BUT...if Weblate .resx generated files use Microsoft ResX Schema 2.0 specification, you could just rename it from .resx to .resw and you have the same file (note: .resw cannot generate any .designer.cs static file, maybe @FlorentRevest has noticed this feature when he edited the en-US xml file).

In short: if you can instruct Weblate to export .resx files that uses ResX Schema 2.0 specification, you can rename file extensions and easily import them into the project, but you also need to edit the .csproj to include new files
